New Entry: General Geolocation (#828)
Codecademy/docs: content/general/concepts/geolocation/geolocation.md

---
Title: 'Geolocation'

> Description: 'Geolocation refers to the process of determining the geographic location of an object, typically through electronic means such as GPS.'

Reviewer: ```suggestion
Description: 'Geolocation refers to the process of determining the geographic location of an object, usually through electronic means such as GPS.'
```

Subjects:
  - 'Computer Science'
  - 'Mobile Development'
Tags:

>   - 'Map'

Reviewer: ```suggestion
  - 'Map'
  - 'Positioning'
```

CatalogContent:
  - 'paths/code-foundations'
  - 'paths/computer-science'
---

> **Geolocation** refers to the process of determining the geographic location of an object, typically through electronic means such as GPS. The most common example of geolocation currently is the location function in various electronic devices such as smartphones.

Reviewer: ```suggestion
**Geolocation** refers to the process of determining the geographic location of an object, usually through electronic means such as GPS. A modern example of geolocation is the location function in various electronic devices such as smartphones.
```

Many modern applications can be given access to a device's geolocation data, so a review app can show restaurants within a few miles, a banking app can show nearby branches, or a game can offer [augmented reality](https://www.codecademy.com/resources/docs/general/augmented-reality) experiences based on where the device is located.
Also, most smartphones have a mapping app that can locate the phone on a street map in order to provide directions to some other location.

> ## Common Types of Geolocation

Reviewer: I don't know if this header accurately summarizes what's going on in the section. This is describing three _"ways"_ of accessing geolocation services; not necessarily "types". What about something like "Using Geolocation Services"?

> There are three main ways a smartphone (and other consumer electronics) can provide geolocation services:

Reviewer: ```suggestion
There are three main ways a smartphone (and other consumer electronic device) can provide geolocation services:
```

- GPS: A device can have a GPS receiver that communicates with the GPS satellite system that uses the positions of the satellites it can communicate with to determine its position on the ground. A disadvantage is that things like heavy tree cover or buildings can interfere with the signal.

> - Wireless: A device connects with several different cell towers in the area and using signal strength and [triangulation](https://en.wikipedia.org/wiki/Triangulation) determines its position on the ground. The disadvantage is this can be inaccurate when there are limited cell towers available, and will not work where there is no cell reception.

Reviewer: If this is referring to [WPS](https://en.wikipedia.org/wiki/Wi-Fi_positioning_system), I'd suggest the following:

```suggestion
- The Wi-Fi Positioning System (WPS): A device connects with several different cell towers in the area and using signal strength and [triangulation](https://en.wikipedia.org/wiki/Triangulation) determines its position on the ground. The disadvantage is this can be inaccurate when there are limited cell towers available, and will not work where there is no cell reception.
```

Author: WPS is distinct. One uses the cellular network, the other uses Wi-Fi hotspots.

---
Title: 'Geolocation'
Description: 'Geolocation refers to the process of determining the geographic location of an object, usually through electronic means such as GPS.'
Subjects:
  - 'Computer Science'
  - 'Mobile Development'
Tags:
  - 'Map'
  - 'Positioning'
CatalogContent:
  - 'paths/code-foundations'
  - 'paths/computer-science'
---

**Geolocation** refers to the process of determining the geographic location of an object, usually through electronic means such as GPS. A modern example of geolocation is the location function in various electronic devices such as smartphones.

> Many modern applications can be given access to a device's geolocation data, so a review app can show restaurants within a few miles/kilometers, a banking app can show nearby branches, or a game can offer [augmented reality](https://www.codecademy.com/resources/docs/general/augmented-reality) experiences based on where the device is located.
> Also, most smartphones have a mapping app that can locate the phone on a street map in order to provide directions to some other location.
> 
> ## Common Methods of Geolocation

Reviewer: ```suggestion
Many modern applications can be given access to a device's geolocation data, so a review app can show restaurants within a few miles/kilometers, a banking app can show nearby branches, or a game can offer [augmented reality](https://www.codecademy.com/resources/docs/general/augmented-reality) experiences based on where the device is located.

Also, most smartphones have a mapping app that can locate the phone on a street map in order to provide directions to some other location.

## Common Geolocation Methods 
```
